doc/animation_widget_article/article.md

```diff
@@ -1,5 +1,13 @@
 # Sprite Sheet Animations in Flutter
 
+### UPDATE 05/27/2020
+
+As for Flame 0.22.0 there is a new way to use Animations and Sprites inside your widget tree.
+
+Flame now includes a widget catalog and inside it you will find `FlameAnimationWidget` and `FlameSpriteWidget`.
```

Comment:
I think if we rename them without the prefix it will look a bit nicer, if there is a clash it is easy to use the named imports in dart anyways? :)